kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -80,13 +80,13 @@
   line-height: 18px;
   text-align: center;
   text-transform: uppercase;
-  color: #4379B1;
+  color: #4379b1;
```

Comment:
stylelint, the css formatter I have been using, lowercases all the colour lettering.